doc/api/module.md

```diff
@@ -217,6 +217,39 @@ added: v22.8.0
 * Returns: {string|undefined} Path to the [module compile cache][] directory if it is enabled,
   or `undefined` otherwise.
 
+### `module.getPackageJSON(startPath[, everything])`
+
+<!-- YAML
+added: VERSION
```

Comment:
What keyword am I suppose to put here? I can't find it in the contributing docs (ex `api-documentation.md`)

Reply:
it should be replaceme all capital.